smart_proxy_dynflow_core.gemspec

```diff
@@ -0,0 +1,38 @@
+# -*- encoding: utf-8 -*-
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'smart_proxy_dynflow_core/version'
+
+Gem::Specification.new do |gem|
+  gem.name          = "smart_proxy_dynflow_core"
+  gem.version       = SmartProxyDynflowCore::VERSION
+  gem.authors       = ['Ivan Nečas']
+  gem.email         = ['inecas@redhat.com']
+  gem.homepage      = "https://github.com/theforeman/smart_proxy_dynflow"
+  gem.summary       = "Dynflow runtime for Foreman smart proxy"
+  gem.description   = <<-EOS
+    Use the Dynflow inside Foreman smart proxy
+  EOS
+
+  gem.files         = Dir['lib/smart_proxy_dynflow_core.rb', 'config/settings.yml.example',
+                          'bundler.d/*', 'lib/smart_proxy_dynflow_core/*', 'LICENSE', 'Gemfile',
```

Comment:
this way I suppose you're missing `lib/smart_proxy_dynflow_core/.rb` (which I see is listed above), why not using the same approach as in smart_proxy_remote_execution_ssh with /**/*? anyway that's minor thing